notebooks/official/model_monitoring/get_started_with_model_monitoring_automl.ipynb

```diff
@@ -31,7 +31,7 @@
         "id": "fsv4jGuU89rX"
```

Comment:
<blockquote>The version of this dataset you use only the fields year, month and day&nbsp;</blockquote><p>The wording is not smooth :D <code>The version of this dataset</code> could be replaced with  </p><blockquote>In  this notebook, you use only the fields year, month and day from the dataset to ....</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
done